graphene/relay/connection.py

```diff
@@ -26,7 +26,7 @@ class EdgeMeta:
 
     edge_name = f"{base_name}Edge"
 
-    edge_bases = [edge_class] if edge_class else [EdgeBase]
+    edge_bases = [edge_class, EdgeBase] if edge_class else [EdgeBase]
```

Comment:
Do we want to add `EdgeBase` to edge_class as a base, or do we just want to put it upon the user to have node and cursor in their definition of edge class? 🙂 Simply not sure if this would be the behavior a user expects 🙂

Reply:
Ideally we don't want to add this here, but we have to do so  in order not to create a breaking change.

The workaround is to create a custom `Connection` that does not use this function and instead doesn't put  `EdgeBase`  into the custom `Edge` bases - hacky, but it works

Reply:
Ahh okay got it 👍 The let's keep it as is 👍